rockpro64-convert-stage-2.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+output_dir=$1
+boot_mapping=$2
+embedded_rootfs_dir=$3
+uboot_backup_dir=${embedded_rootfs_dir}/opt/backup/uboot
+build_log=$output_dir/build.log
+
+[ ! -f $output_dir/boot.vfat ] && \
+    { log "Error: extracted boot partition not found. Aborting."; exit 1; }
+[ ! -d "${embedded_rootfs_dir}" ] && \
+    { log "Error: embedded content not mounted."; exit 1; }
```

Comment:
It would be nice to check also that `/dev/mapper/${boot_mapping}` exists

Reply:
Fixed